spec/views/authors/new.html.erb_spec.rb

```diff
@@ -0,0 +1,52 @@
+RSpec.describe "authors/new" do
+  before do
+    assign(:author, author)
+  end
+
+  describe "a blank form" do
+    let(:author) { Author.new }
+
+    before(:each) { render }
+
+    it "does not render an error list" do
+      assert_select "#error_explanation", false
```

Comment:
they've never seen `assert_select`. We generally do view layer tests with Capybara